runtime/parachains/src/initializer.rs

```diff
@@ -156,6 +173,31 @@ impl<T: Trait> Module<T> {
 		scheduler::Module::<T>::initializer_on_new_session(&notification);
 		inclusion::Module::<T>::initializer_on_new_session(&notification);
 	}
+
+	/// Should be called when a new session occurs. Buffers the session notification to be applied
+	/// at the next block. If `queued` is `None`, the `validators` are considered queued.
+	fn on_new_session<'a, I: 'a>(
+		_changed: bool,
+		session_index: sp_staking::SessionIndex,
+		validators: I,
+		queued: Option<I>,
+	)
+		where I: Iterator<Item=(&'a T::AccountId, ValidatorId)>
```

Comment:
It may not matter, but `IntoIterator` bounds are sometimes more convenient than `Iterator` bounds.